bundles/org.openhab.binding.nuki/src/main/java/org/openhab/binding/nuki/internal/dataexchange/NukiApiServlet.java

```diff
@@ -48,34 +47,28 @@
  *
  * @author Markus Katter - Initial contribution
  * @contributer Christian Hoefler - Door sensor integration
+ * @contributer Jan Vybíral
```

Comment:
Please also add what you have changed in short.
```suggestion
 * @author Christian Hoefler - Door sensor integration
 * @author Jan Vybíral
```